frontend/src/components/cluster/KubeConfigLoader.tsx

```diff
@@ -181,7 +181,7 @@ function KubeConfigLoader() {
         }
         setFileContent(doc);
       } catch (err) {
-        setError(t('translation|Load a valid kubeconfig'));
+        setError(t('translation|Unsupported encoding: Please use UTF-8 encoding'));
```

Comment:
There is the possibility that the error is related to encoding, but there are other possibilities, including the one we are directly throwing, on like 180 above this catch block.
I think we should do two things:
1. replace the `Invalid kubeconfig file` message from the error by something translatable. We can even throw two different messages, since we are checking for two different things there (existence of `clusters` and existence of `contexts`).
2. Use the `err.message` to set the error. This way, no matter what the error is, we know it will be more related to what actually happened.

Reply:
Made the changes